README.md

```diff
@@ -84,33 +88,22 @@ The ol package contains a src/ folder with JSDoc annotated sources. TypeScript c
   }
 }
 ```
-
 </details>
-<details><summary>tsconfig.json</summary>
 
-```json
+## TypeScript support
+
+When authoring in TypeScript, we recommend you try out the types that are included in the `ol` package. To use these types, make sure you have the following entry for `@types/ol` in your `package.json`'s `devDependencies` section:
+```
 {
-  "compilerOptions": {
-    "allowJs": true,
-    "baseUrl": "./",
-    "paths": {
-      "ol": ["node_modules/ol/src"],
-      "ol/*": ["node_modules/ol/src/*"]
-    }
-  },
-  "include": [
-    "**/*.ts",
-    "node_modules/ol/**/*"
-  ],
-  "typeAcquisition": {
-    "exclude": ["ol"]
+  ...
+  "devDependencies": {
+    ...
+    "@types/ol": "file:node_modules/ol/types",
+    ...
   }
 }
 ```
-
-</details>
-
-TypeScript users may want to use a [third-party types package](https://github.com/hanreev/types-ol) instead.
+These are auto-generated with the TypeScript compiler, and will be the default in future versions. Alternatively, you can use third-party types from [Definitely Typed](https://definitelytyped.org) (`npm install @types/ol`).
```

Comment:
Last time I checked (maybe around december last year) the `@hanreev/types-ol` were providing better typing then the `@types/ol`. Maybe mentioning both?

Reply:
```suggestion
These are auto-generated with the TypeScript compiler, and will be the default in future versions. Alternatively, you can use third-party types from [Definitely Typed](https://definitelytyped.org) (`npm install @types/ol`) or from [hanreev/types-ol](https://github.com/hanreev/types-ol).
```